clang/lib/Sema/SemaChecking.cpp

```diff
@@ -11647,6 +11647,29 @@ static void DiagnoseFloatingImpCast(Sema &S, Expr *E, QualType T,
   }
 }
 
+static void CheckCommaOperand(Sema &S, Expr *E, QualType T, SourceLocation CC,
+                              bool Check) {
```

Comment:
`Check` is pretty weird here... I'd almost suggest something like, `isResultOperand` or something?  But doing away with thsi function entirely wouldn't harm readability.

Reply:
It's "Check" because it emits diagnostics, same as other functions in the file.

Reply:
Ah, sorry, I meant the bool variable :D  It is pretty ambiguous/clumsy IMO.

Reply:
Renamed